src/PlotRecipes/ElectricField.jl

```diff
@@ -5,7 +5,8 @@
                     φ = missing,
                     z = missing,
                     contours_equal_potential=false,
-                    full_det = false ) where {T}
+                    full_det = false,
+                    all_point_types_bits = false) where {T}
```

Comment:
I don't like that we need to add keywords to plot recipes where they don't make sense.
(`all_point_types_bits` in `ElectricField`??)

Reply:
Sounds good!